build_scripts/windows/scripts/build.cmd

```diff
@@ -43,6 +48,7 @@ set ARTIFACTS_DIR=%~dp0..\artifacts
 mkdir %ARTIFACTS_DIR%
 set TEMP_SCRATCH_FOLDER=%ARTIFACTS_DIR%\cli_scratch
 set BUILDING_DIR=%ARTIFACTS_DIR%\cli
+set ZIP_DIR=%ARTIFACTS_DIR%\Azure CLI
```

Comment:
A folder name with space (`Azure CLI`) causes lots of trouble when the full path is being invoked from PowerShell.

For example, the MSI installs the Azure CLI executable at `C:\Program Files\Microsoft SDKs\Azure\CLI2\wbin\az.cmd`.

It can't be directly executed because of the space in the path:

```
> C:\Program Files\Microsoft SDKs\Azure\CLI2\wbin\az.cmd
C:\Program: The term 'C:\Program' is not recognized as a name of a cmdlet, function, script file, or executable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
```

If we quote it, PowerShell treats it as a string and echoes it:

```
> "C:\Program Files\Microsoft SDKs\Azure\CLI2\wbin\az.cmd"
C:\Program Files\Microsoft SDKs\Azure\CLI2\wbin\az.cmd
```

The correct way to invoke the executable with its full path is:

```
> & "C:\Program Files\Microsoft SDKs\Azure\CLI2\wbin\az.cmd"

Welcome to Azure CLI!
---------------------
Use `az -h` to see available commands or go to https://aka.ms/cli.
```

Of course, we don't what such complex way of invoking `az.cmd`.